mkdocs.yaml

```diff
@@ -1,6 +1,6 @@
 site_name: Autoware Documentation
 site_url: https://autowarefoundation.github.io/autoware-documentation
-repo_url: https://github.com/autowarefoundation/autoware-documentation
+repo_url: https://github.com/autowarefoundation/autoware
 edit_uri: edit/main/docs/
```

Comment:
I forgot about `edit_uri`. It doesn't work well.

Reply:
It worked.
![image](https://user-images.githubusercontent.com/31987104/168441656-b346353c-5752-4f62-8e4a-0a209a5061a5.png)